src/transformers/models/pegasus/tokenization_pegasus.py

```diff
@@ -80,7 +80,6 @@ class PegasusTokenizer(PreTrainedTokenizer):
     """
     vocab_files_names = VOCAB_FILES_NAMES
 
-    offset = 103  # entries 2 - 104 are only used for pretraining
```

Comment:
While it seems to be very bad practice to adapt an existing tokenizer file to add a new model, I strongly advocate to do so here. The reasons are the following:

- The official tokenizer is a pegasus tokenizer, see [here](https://github.com/google-research/bigbird/blob/master/bigbird/vocab/pegasus.model).
- The previous implementation was very restrictive by forcing `offset` to be of a certain value, which is too specific for a general PEGASUS like model IMO
- The required adaptions to the existing tokenizer file are ok IMO: We add the possibility of the `mask_sent_token` to be `None` & allow `offsets` to be whatever value fits.

Reply:
@LysandreJik @sgugger

Reply:
That's fine by me.

Reply:
Yes, works for me too